db/seeds.rb

```diff
@@ -9,109 +9,72 @@
 )
 Rails.logger.info "✔ States initialized"
 
-# Initialize cities
-cities = [
-  ["Amsterdam",       555],
-  ["Bali",            314],
-  ["Barcelona",       506],
-  ["Belize",          14],
-  ["Belo Horizonte",  22],
-  ["Berlin",          1978],
-  ["Bordeaux",        615],
-  ["Brasilia",        134],
-  ["Brussels ⭐",      732],
-  ["Buenos Aires",    268],
-  ["Casablanca",      88],
-  ["Chengdu",         64],
-  ["Cologne",         52],
-  ["Dubai",           35],
-  ["Emil",            124],
-  ["For Business",    87],
-  ["Istanbul",        30],
-  ["Kyoto",           18],
-  ["Lausanne",        170],
-  ["Lille",           438],
-  ["Lima",            53],
-  ["Lisbon",          829],
-  ["London",          1882],
-  ["Lyon",            483],
-  ["Madrid",          162],
-  ["Malmö",           21],
-  ["Marseille",       565],
-  ["Martinique",      42],
-  ["Mauritius",       117],
-  ["Medellín",        71],
-  ["Melbourne",       386],
-  ["Mexico",          231],
-  ["Milan",           137],
-  ["Montréal",        534],
-  ["Munich",          219],
-  ["Nantes",          303],
-  ["Nice",            122],
-  ["Paris",           3108],
-  ["Porto",           21],
-  ["Remote",          417],
-  ["Rennes",          67],
-  ["Rio de Janeiro",  427],
-  ["Santiago",        98],
-  ["São Paulo",       746],
-  ["Seine et Marne",  11],
-  ["Shanghai",        386],
-  ["Shenzhen",        24],
-  ["Singapore",       309],
-  ["Stockholm",       51],
-  ["Tel Aviv",        158],
-  ["Tokyo",           592],
-  ["Zurich",          30]
-].map do |city|
-  { name: city[0], size: city[1] }
+# Initialize cities & batch
+require "csv"
+CSV.foreach("app/assets/batch_map.csv") do |row|
+  city = City.find_by(name: row[1])
+  city = City.create!(name: row[1]) if city.nil?
```

Comment:
Cas d'école pour `find_or_create`.
```suggestion
  city = City.find_or_create_by!(name: row[1])
```